src/portal/src/app/project/webhook/webhook.component.html

```diff
@@ -0,0 +1,53 @@
+<div class="row">
+  <div *ngIf="!showCreate">
+    <div>
+        <div class="row flex-items-xs-between rightPos">
+            <div class="flex-xs-middle option-left">
+              <div>
+                <span style="font-weight: bold;">Webhook endpoint</span>: {{endpoint}}
+                <button class="btn btn-link" (click)="openAddWebhookModal()">{{'WEBHOOK.EDIT_BUTTON' | translate}}</button>
+              </div>
+              <div [ngSwitch]="isEnabled">
+                <button *ngSwitchCase="false" class="btn btn-link" (click)="switchWebhookStatus(true)">{{'WEBHOOK.ENABLED_BUTTON' | translate}}</button>
+                <button *ngSwitchCase="true" style="color: #cf4c23;" class="btn btn-link" (click)="switchWebhookStatus(false)">{{'WEBHOOK.DISABLED_BUTTON' | translate}}</button>
+              </div>
+            </div>
+        </div>
+    </div>
+    <div class="col-lg-12 col-md-12 col-sm-12 col-xs-12 datagrid-margin-top">
+      <clr-datagrid [clrDgLoading]="loading">
+          <clr-dg-column>{{'WEBHOOK.TYPE' | translate}}</clr-dg-column>
+          <clr-dg-column>{{'WEBHOOK.STATUS' | translate}}</clr-dg-column>
+          <clr-dg-column>{{'WEBHOOK.CREATED' | translate}}</clr-dg-column>
+          <clr-dg-column>{{'WEBHOOK.LAST_TRIGGERED' | translate}}</clr-dg-column>
+          <clr-dg-row *ngFor="let item of lastTriggers">
```

Comment:
No paging for lastTriggers ?

Reply:
replace "*ngFor" with "*clrDgItem", if you want to page by datagrid component 。And normally, set [clrDgPageSize] ="15"

Reply:
That's right.It's not ready for backend.

Reply:
FYI
![image](https://user-images.githubusercontent.com/30999793/62764652-fcd7d080-bac0-11e9-8a81-493a9cf96dfa.png)
Please modify.